releasenotes/notes/admissioncontroller-prepend-env-vars-8d88283457c55d9b.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    when injecting env vars with the admission controller, env
+    vars are now prepended instead of appended, meaning that 
+    kubernetes [dependent environment variables](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/)
+    can now depend on said injected vars. 
```

Comment:
```suggestion
    When injecting env vars with the admission controller, env
    vars are now prepended instead of appended, meaning that 
    Kubernetes [dependent environment variables](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/)
    can now depend on these injected vars. 
```